package.json

```diff
@@ -108,6 +110,7 @@
     "test:e2e:components-react": "yarn --cwd ./packages/components-react test:e2e",
     "test:e2e:components-vue": "yarn --cwd ./packages/components-vue test:e2e",
     "test:e2e:storefront": "yarn --cwd ./packages/storefront test:e2e",
+    "test:e2e:crawler": "yarn --cwd ./packages/crawler test:e2e",
```

Comment:
Please stick to the same order. The `start` command was listed after `components-vue`, but here it is after `storefront`.